data/tmpl/values/content3.html

```diff
@@ -32,7 +32,11 @@
             </label>
           </li>
         </ul>
-        <button class="button">Tell Me</button>
+        <button id="tell_me" class="button">Tell Me</button>
+        <section id="private_browsing" class="hidden">
```

Comment:
Yep, thats exactly what I was thinking with the fade. I talked to @chrismore a bit about UX stuff on the sidebars, and since Matt is overloaded with work for the time being, we will revisit it all in London when we can work quickly in person!